csv/phases.csv

```diff
@@ -0,0 +1,55 @@
+Phase.TRAVERSE
```

Comment:
Could you clear csv/phases.csv, csv/datastore.csv, and rpi_to_gui.simulation.csv? A lil context on this, but we have our code write to these csv files when the code is run to pass information between our engine and GUI modules. This happens whenever our code is run, including tests. However, we usually want to keep them empty on our main branch for storage reasons. Just a select all + delete for the files

Reply:
Sammy made a great comment.